### DISCUSSION TITLE
how to know the machine learning is working

### DISCUSSION THREAD

**Discussion by 18677386630** [immich-app/immich#7984] [Q&A] [answered]
### The bug

I don't know why the FACE DETECTION doesn't working
![Screenshot_2024-03-15-20-15-09-747_app alextran immich](https://github.com/immich-app/immich/assets/130483453/cbc773a1-e04c-408b-b42b-8d4cdeb8a478)
![Screenshot_2024-03-15-20-14-57-462_alook browser](https://github.com/immich-app/immich/assets/130483453/eae2a1e4-a050-49cc-b05f-db97ce90dbda)

### The OS that Immich Server is running on

armbian

### Version of Immich Server

v1.98.2

### Version of Immich Mobile App

v1.98.2

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
example
```

### Your .env content

```Shell
example
```

### Reproduction steps

```bash
1.no bug it is my question
2.
3.
...
```

**Comment by schuhbacca** [comment]
If you are based in China the model for ML is blocked I believe. See here: https://github.com/immich-app/immich/discussions/6417#discussioncomment-8139563

**Comment by 18677386630** [comment]
> If you are based in China the model for ML is blocked I believe. See here: [#6417 (comment)](https://github.com/immich-app/immich/discussions/6417#discussioncomment-8139563)

thanks for your help，I will test it
maybe the problem is poetry.can you teach me how to confirm it

**Comment by schuhbacca** [comment]
> thanks for your help，I will test it maybe the problem is poetry.can you teach me how to confirm it

I think you can look at the machine learning container and see if it has any errors. Failed to download or something like that.

**Comment by aviv926** [comment, marked as answer]
![image](https://github.com/immich-app/immich/assets/51673860/bf7a898f-c478-4182-98c6-0a08e2b1d65a)

**Reply by 18677386630** [reply]
thanks，I got it
